ballerina/tests/sql_rainier_generated_client.bal

```diff
@@ -7,14 +23,14 @@ const BUILDING = "buildings";
 const DEPARTMENT = "departments";
 const ORDER_ITEM = "orderitems";
 
-public client class RainierClient {
-    *AbstractPersistClient;
+public client class SQLRainierClient {
+    *AbstractPersistSQLClient;
```

Comment:
We need this change in the generated client? @kalaiyarasiganeshalingam

Reply:
Not needed